pwndbg/pdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module serves as utility for the developers to get pdb debugging session right, whenever standard:
+    import pdb; pdb.set_trace()
+Can't do its job properly (that happens often as gdb does some magic with stdout/stderr).
+
+Thanks to Dmoreno: http://stackoverflow.com/questions/17074177/how-to-debug-python-cli-that-takes-stdin
+
+Usage:
+    import pwndbg.pdb
+
+Importing the module will print out pretty message on what to do next.
+"""
+import functools
+import pdb
```

Comment:
Hm.. the description from stdio.py says:
```
Provides functionality to circumvent GDB's hooks on sys.stdin and sys.stdout
which prevent output from appearing on-screen inside of certain event handlers.
```

Can we use it to prevent warnings in gdb's `find` command that we use through `gdb.execute`?